opencti-platform/opencti-front/src/private/components/data/connectors/ConnectorWorks.tsx

```diff
@@ -0,0 +1,376 @@
+                  <Grid item={true} xs={6}>
+                    <Typography
+                      variant="h3"
+                      gutterBottom={true}
+                      style={{ marginTop: 20 }}
+                    >
+                      {t_i18n('Work end time')}
+                    </Typography>
+                    {work.completed_time ? nsdt(work.completed_time) : '-'}
+                  </Grid>
+                </Grid>
+              </Grid>
+              <Grid item={true} xs={5}>
+                <Grid container={true} spacing={3}>
+                  <Grid item={true} xs={6}>
+                    <Typography variant="h3" gutterBottom={true}>
+                      {t_i18n('Operations completed')}
+                    </Typography>
+                    <span className={classes.number}>
+                      {work.status === 'wait'
+                        ? '-'
+                        : tracking?.import_processed_number ?? '-'}
+                    </span>
+                  </Grid>
+                  <Grid item={true} xs={6}>
+                    <Typography variant="h3" gutterBottom={true}>
+                      {t_i18n('Total number of operations')}
+                    </Typography>
+                    <span className={classes.number}>
+                      {work.status === 'wait'
+                        ? '-'
+                        : tracking?.import_expected_number ?? '-'}
+                    </span>
+                  </Grid>
+                  <Grid item={true} xs={12}>
+                    <Typography variant="h3" gutterBottom={true}>
+                      {t_i18n('Progress')}
+                    </Typography>
+                    <LinearProgress
+                      classes={{ root: classes.progress }}
+                      variant="determinate"
+                      value={
+                        // eslint-disable-next-line no-nested-ternary
+                        tracking && !!tracking.import_expected_number && !!tracking.import_processed_number
+                          ? Math.round((tracking.import_processed_number / tracking.import_expected_number) * 100)
+                          : 0
+                      }
+                    />
+                  </Grid>
+                </Grid>
+              </Grid>
+              <Button
+                style={{ position: 'absolute', right: 10, top: 10 }}
+                variant="contained"
+                color="secondary"
+                onClick={() => handleOpenErrors(work.errors ?? [])}
+                size="small"
+              >
+                {work.errors?.length} {t_i18n('errors')}
+              </Button>
+              <Button
+                variant="outlined"
+                style={{ position: 'absolute', right: 10, bottom: 10 }}
+                onClick={() => handleDeleteWork(work.id)}
+                size="small"
+              >
+                <Delete fontSize="small" />
+                &nbsp;&nbsp;{t_i18n('Delete')}
```

Comment:
Can we delete it ? `&nbsp;&nbsp;`

Reply:
Obviously. it was used to put a space between the icon and the text in the button. I replace it with the `startIcon` property of the button